doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_reg_spec.rst

```diff
@@ -6,6 +6,23 @@ Light Lightness Control spec regulator
 This module implements the PI lightness regulator specified in the Bluetooth® mesh model specification.
 Initialize using :c:macro:`BT_MESH_LIGHT_CTRL_REG_SPEC_INIT`.
 
+The illuminance regulator takes the specified target illuminance level as the reference level and compares it to sensor data from an external illuminance sensor.
```

Comment:
We are not mentioning the illuminance regulator before this, only the PI lightness regulator. Are these the one and the same, or are we referring to something else?

Reply:
I think here is the definition: https://github.com/nrfconnect/sdk-nrf/blame/691dc8fae2a85f411cc0a0f0b989479c8bcf945e/doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst#L358

Reply:
It seems like the illuminance regulator was the name of the internal lightness regulator, right?
So now we are operating with several names:
- Light Lightness Control default/standard regulator
- PI lightness regulator
- Illuminance regulator
Just within a few paragraphs.
We should be careful with the names. Should we keep illuminance regulator as a name. Or should we detach?

Reply:
I think it looks much better. More cohesive :)